eve/main.yml

```diff
@@ -77,6 +78,68 @@ stages:
             - SHA256SUM
             - product.txt
 
+  buildprev:
+    worker:
+      type: kube_pod
+      path: eve/workers/pod-builder/pod.yaml
+      images:
+        docker-builder: eve/workers/pod-builder
+    steps:
+      - ShellCommand:
+          name: Wait for Docker daemon to be ready
+          command: |
+            bash -c '
+            for i in {1..150}
+            do
+              docker info &> /dev/null && exit
+              sleep 2
+            done
+            echo "Could not reach Docker daemon from buildbot worker" >&2
+            exit 1'
+          haltOnFailure: true
+      - SetPropertyFromCommand:
+          name: set product_version_prev
+          property: product_version_prev
+          command: >
+              major=$(echo "%(prop:product_version)s" | cut -d'.' -f1) &&
```

Comment:
Details: prefer the herestring syntax rather than using `echo |` which is ugly.
```
major=$(cut -d'.' -f1 <<< "%(prop:product_version)s")
```
And, btw `&&` are useless since there is no chance that your cut command will fail.

Reply:
ups will fix this in the next PRs